junit-jupiter-engine/src/test/java/org/junit/jupiter/engine/discovery/DiscoverySelectorResolverTests.java

```diff
@@ -287,16 +309,31 @@ void methodResolutionByUniqueIdWithParams() {
 		List<UniqueId> uniqueIds = uniqueIds();
 		assertThat(uniqueIds).contains(uniqueIdForClass(HerTestClass.class));
 		assertThat(uniqueIds).contains(uniqueIdForMethod(HerTestClass.class, "test7(java.lang.String)"));
+		// @formatter:off
+		assertThat(listener.stream()
+				.filter(logRecord -> JavaElementsResolver.class.getName().equals(logRecord.getLoggerName()))
+		).isEmpty();
```

Comment:
You should be able to remove the filter by using `org.junit.platform.commons.logging.LogRecordListener.stream(Class<?>)`.

Reply:
👍